sdk/machinelearningservices/Azure.ResourceManager.MachineLearning/src/Generated/Models/AmlComputeProperties.Serialization.cs

```diff
@@ -169,15 +169,22 @@ void IJsonModel<AmlComputeProperties>.Write(Utf8JsonWriter writer, ModelReaderWr
             }
             if (Optional.IsDefined(PropertyBag))
             {
-                writer.WritePropertyName("propertyBag"u8);
+                if (PropertyBag != null)
```

Comment:
Modeler4 emit it as nullable, so the change is expected.

Same reason for all following serialization changes